pkg/targets/adapter/azuresentineltarget/adapter.go

```diff
@@ -0,0 +1,143 @@
+/*
+Copyright 2022 TriggerMesh Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package azuresentineltarget
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/azure/auth"
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+	"github.com/google/uuid"
+	"go.uber.org/zap"
+	pkgadapter "knative.dev/eventing/pkg/adapter/v2"
+	"knative.dev/pkg/logging"
+
+	targetce "github.com/triggermesh/triggermesh/pkg/targets/adapter/cloudevents"
+)
+
+type azuresentineltargetAdapter struct {
+	client         *http.Client
+	clientID       string
+	tenantID       string
+	azureCreds     string
+	subscriptionID string
+	resourceGroup  string
+	workspace      string
+	clientSecret   string
+
+	// sink     string
+	replier  *targetce.Replier
+	ceClient cloudevents.Client
+	logger   *zap.SugaredLogger
+}
+
+// EnvAccessorCtor for configuration parameters
+func EnvAccessorCtor() pkgadapter.EnvConfigAccessor {
+	return &envAccessor{}
+}
+
+// NewTarget adapter implementation
+func NewTarget(ctx context.Context, envAcc pkgadapter.EnvConfigAccessor, ceClient cloudevents.Client) pkgadapter.Adapter {
+	env := envAcc.(*envAccessor)
+	logger := logging.FromContext(ctx)
+
+	replier, err := targetce.New(env.Component, logger.Named("replier"),
+		targetce.ReplierWithStatefulHeaders(env.BridgeIdentifier),
+		targetce.ReplierWithStaticResponseType("io.triggermesh.azuresentineltarget.response"),
+		targetce.ReplierWithPayloadPolicy(targetce.PayloadPolicy(env.CloudEventPayloadPolicy)))
+	if err != nil {
+		logger.Panicf("creating CloudEvents replier: %v", err)
```

Comment:
I'd change this to `logger.Panicf("Error creating CloudEvents replier: %v", err)` to be more standard with other components.

Reply:
I find an error message stating that its an error message to be a bit redundant. But i will conform : )